dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.References.targets

```diff
@@ -1,23 +0,0 @@
-<!--
-  Copyright (c) Microsoft Corporation. Licensed under the MIT License
--->
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-  <PropertyGroup>
-    <_MrtCoreRuntimeIdentifier Condition="'$(Platform)' == 'Win32'">x86</_MrtCoreRuntimeIdentifier>
-    <_MrtCoreRuntimeIdentifier Condition="'$(Platform)' != 'Win32'">$(Platform)</_MrtCoreRuntimeIdentifier>
-    <_MrtCoreDllDir>$(MSBuildThisFileDirectory)..\runtimes\win10-$(_MrtCoreRuntimeIdentifier)\native\</_MrtCoreDllDir>
-
-    <_AddMrtCoreAssembliesToReferenceCopyLocalPaths Condition="'$(_AddMrtCoreAssembliesToReferenceCopyLocalPaths)'=='' AND '$(Platform)' == 'AnyCPU'">false</_AddMrtCoreAssembliesToReferenceCopyLocalPaths>
-    <_AddMrtCoreAssembliesToReferenceCopyLocalPaths Condition="'$(_AddMrtCoreAssembliesToReferenceCopyLocalPaths)'=='' AND '$(ProjectReunionFrameworkPackage)' == 'true'">false</_AddMrtCoreAssembliesToReferenceCopyLocalPaths>
-    <_AddMrtCoreAssembliesToReferenceCopyLocalPaths Condition="'$(_AddMrtCoreAssembliesToReferenceCopyLocalPaths)'=='' AND '$(MSBuildProjectExtension)' == '.wapproj'">true</_AddMrtCoreAssembliesToReferenceCopyLocalPaths>
```

Comment:
@huichen123 is there a better condition to use than '$(ProjectReunionFrameworkPackage)' == 'true' ?  I'd like to keep ProjectReunionFrameworkPackage normally undefined, and only explicitly set to 'false' for InApp mode.  But I don't want that condition used here, or it will collide with the InApp mode support.

Reply:
if we want that property to be implicit, we have to introduce other property explicitly set for inapp mode.

Reply:
It sounds like you need another property in any case, to distinguish a third condition (MRT inner loop testing) from Project Reunion deployment, whether in Framework Package or InApp mode.  For the related WinUI PR, I used the ProjectReunionWinUI property for that purpose.  Should I just create something like that here?  ProjectReunionMRT?